src/backends/github/API.js

```diff
@@ -156,18 +156,33 @@ export default class API {
   }
 
   readFile(path, sha, branch = this.branch) {
-    const cache = sha ? LocalForage.getItem(`gh.${ sha }`) : Promise.resolve(null);
-    return cache.then((cached) => {
-      if (cached) { return cached; }
-
+    if (sha) {
```

Comment:
@Benaiah I'm assuming if we know the SHA we should fetch by SHA first. Is that a problem at all?

Reply:
Unless there's some benefit to using the Contents API, e.g. differences in the response object, fetching by SHA make sense to me.